simqueue/views.py

```diff
@@ -110,9 +110,12 @@ def copy_datafiles_to_storage(request, target, job_id):
 def copy_datafiles_to_collab_storage(request, job, local_dir, relative_paths):
 
     # upload local files to collab storage
-    from bbp_client.oidc.client import BBPOIDCClient
-    from bbp_client.document_service.client import Client as DocClient
-    import bbp_services.client as bsc
+    #from bbp_client.oidc.client import BBPOIDCClient
```

Comment:
There seem to be some missing changes here - i.e. the changes from the `feature/without-epfl-modules` branch